plugin/config/import/pluginfieldscartridgecartridgecustomfield.php

```diff
@@ -0,0 +1,40 @@
+<?php
+$additionalFields = [
+    [
+        'name'     => 'items_id',
+        'old_name' => 'id',
+    ],
+    [
+        'name' => 'itemtype',
+    ],
+    [
+        'name' => 'plugin_fields_containers_id',
+    ]
+];
+
+$fieldMap = json_decode(file_get_contents(PLUGIN_ISERVICE_DIR . '/install/customfields/cartridge_customfields.json'), true);
+
+$fieldsContainer = new PluginFieldsContainer();
+$fieldsContainer->getFromDBByCrit(['name' => 'cartridgecustomfield']);
+$fieldsContainerId = $fieldsContainer->getField('id');
+
+if (empty($fieldsContainerId) || $fieldsContainerId == NOT_AVAILABLE) {
```

Comment:
Try to avoid using `==`, or `!=` where possible (it could lead to errors)

```suggestion
if (empty($fieldsContainerId) || $fieldsContainerId === NOT_AVAILABLE) {
```